### ISSUE TITLE
The build fails

### ISSUE THREAD

**Issue by ryonakano** [panosx2/brightness#72]
```
[1/3] Compiling Vala source ../src/Dimmer.vala.
FAILED: com.github.panosx2.brightness@exe/src/Dimmer.c
valac -C --debug --debug --pkg gtk+-3.0 --color=always --directory com.github.panosx2.brightness@exe --basedir ../ ../src/Dimmer.vala
../src/Dimmer.vala:23.7-23.18: error: The namespace name `AppIndicator' could not be found
using AppIndicator;
      ^^^^^^^^^^^^
../src/Dimmer.vala:25.23-25.31: error: The type name `Indicator' could not be found
public class Dimmer : Indicator {
                      ^^^^^^^^^
../src/Dimmer.vala:28.19-28.30: error: The symbol `AppIndicator' could not be found
    public static AppIndicator.Indicator indicator;
                  ^^^^^^^^^^^^
Compilation failed: 3 error(s), 0 warning(s)
ninja: build stopped: subcommand failed.
```

The CI also fails with the same error.

**Comment by panosx2** [comment]
@ryonakano Yeah i know. Some library relevant to `AppIndicator` misses but i don't know which one.

**Comment by ryonakano** [comment]
@panosx2 Ah I could build the app by adding `dependency('appindicator3-0.1')` in the `meson.build` file!

However, with this fix either the app nor the indicator isn't shown on elementary OS Juno, because [elementary (and GNOME) dropped out Application indicators](https://github.com/elementary/wingpanel/issues/96#issuecomment-401407354) (which is **deprecated**) from the OS. If we want to show them, we'll need to install [this patched indicator-applications debian file](https://github.com/mdh34/elementary-indicators/releases) and [`wingpanel-indicator-namarupa`](https://github.com/donadigo/wingpanel-indicator-namarupa) (which are unofficial). By installing them, the app and the indicator is shown finally like this:

![2019-02-04 20 50 47](https://user-images.githubusercontent.com/26003928/52206853-d7260080-28be-11e9-8a5f-0ab6b31d2421.png)

I think a good way to create an indicator is to use [Wingpanel.Indicator](https://github.com/elementary/wingpanel/blob/master/sample/SampleIndicator.vala) in `libwingpanel-2.0-dev`. If you use this, any extra operation described above should not be needed.

**Comment by ryonakano** [comment]
Closing this issue since the build now succeeds. In addition to that, that the build fails and that the app and indicator are not shown are not the same topic.
